src/interfaces/IMetaMorphoV1_1.sol

```diff
@@ -70,9 +70,6 @@ interface IMetaMorphoV1_1Base {
     function withdrawQueueLength() external view returns (uint256);
 
     /// @notice Stores the total assets managed by this vault when the fee was last accrued.
-    /// @dev May be greater than `totalAssets()` due to removal of markets with non-zero supply or socialized bad debt.
```

Comment:
> due to removal of markets with non-zero supply

isn't this still true?

Reply:
I don't think [so](https://github.com/morpho-org/metamorpho-v1.1/blob/5a0717b2c863060ff452ca324716083fe6d09374/src/MetaMorphoV1_1.sol#L378).

I remember that we discussed this

Reply:
See [here](https://github.com/morpho-org/metamorpho-v1.1/pull/20) for more detail on why forced removed markets behave the same as bad debt realization with respect to the values of totalAssets and lastTotalAssets